Source/SwiftLintFramework/Extensions/File+SwiftLint.swift

```diff
@@ -126,3 +126,17 @@ extension File {
         lines = contents.lines()
     }
 }
+
+extension File {
+    internal func ruleEnabledViolatingRanges(violatingRanges: [NSRange], forRule rule: Rule)
+        -> [NSRange] {
+        let fileRegions = self.regions()
```

Comment:
Please omit `self` unless required, as outlined in GitHub's Swift style guide: https://github.com/github/swift-style-guide#only-explicitly-refer-to-self-when-required